versioneer.py

```diff
@@ -560,15 +562,15 @@ def git_get_keywords(versionfile_abs):
         f = open(versionfile_abs, "r")
         for line in f.readlines():
             if line.strip().startswith("git_refnames ="):
-                mo = re.search(r'=\s*"(.*)"', line)
+                mo = re.search('=\\s*"(.*)"', line)
```

Comment:
These regexes still look good

Reply:
Side question: did raw strings go out of style at some point?

Reply:
Huh. We might also want to exclude `conf.py` from sphinx if we're excluding vendored files. This becomes a bit annoying in general to address all these cases on a repo-by-repo basis.

Reply:
It's possible `black` could fix up `conf.py` with a single go, but I wouldn't mind just adding it to ignore for an easier solution

Reply:
I'm ok to generally omit vendored files, since I'm not at all certain that I'm fixing these things the same way in every repo.